src/index.ts

```diff
@@ -205,7 +206,7 @@ export class CdktfProviderProject extends cdk.JsiiProject {
     });
     const upgradeScript = new CheckForUpgradesScriptFile(this, {
       providerVersion,
-      fqproviderName,
+      fqproviderName: fqproviderName,
```

Comment:
This can also stay, the old variant is a short-cut for the new line you wrote

Reply:
👍